descqa/SizeStellarMassLuminosity.py

```diff
@@ -26,7 +27,6 @@ class SizeStellarMassLuminosity(BaseValidationTest):
     _ARCSEC_TO_RADIAN = np.pi / 180. / 3600.
 
     def __init__(self, **kwargs):
-        #pylint: disable=W0231
```

Comment:
add back `#pylint: disable=W0231` to make linter happy.

Reply:
Does this line consider as a comment?

Reply:
Yes, but linter reads this comment and disables this particular warning so that it can pass travis CI build.